src/main/java/org/fundacionjala/trello/pages/board/BoardPage.java

```diff
@@ -72,4 +75,11 @@ public void permanentlyDelete() {
         action.click(addListIcon);
         return new ListForm(driver);
     }
+
+    public FormPage<?> updateList(final String listName) {
+        String getList = String.format(GET_LIST, listName);
+        WebElement actualList = driver.findElement(By.xpath(getList));
+        action.click(actualList);
+        return new ListUpdateForm(driver);
```

Comment:
I try to do the update in me same object but there are many diferent locators an actions, so I create UpdateListForm to manage this locator and actions, because I dont like the idea to use many if() to manage this problem, I prefer other object